core-web/libs/portlets/dot-ema/src/lib/shared/consts.ts

```diff
@@ -44,3 +44,5 @@ export const DEFAULT_PERSONA: DotPersona = {
     hasLiveVersion: false,
     modUser: 'system'
 };
+
+export const PAGINATOR_ITEMS_PER_PAGE = 10;
```

Comment:
We don't need this here, we can encapsulate this in the component itself, also, make sure it mention "palette" so is obvious that is for that comp.